### PR TITLE
Change project name from SMPS to SMP

### LINKED ISSUE #80: Change project name from SMPS to SMP
**Current State:**
SMP is shorter and gets people less tripped up. It doesn't need "Specification" in the title. Many people will already add the word _specification_ as a descriptor when describing SMPS, so the "S" often becomes redundant.

**Desired State:**
- [ ] Change this repository's name (drop "specification")
- [ ] Remove the "S" in SMPS in the logo.
- [ ] Update elsewhere....?

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by JubayerJoy**
Signed-off-by: Jubayer Abdullah Joy jubayerjoycse@gmail.com

**Description**
**Current State:**
SMP is shorter and gets people less tripped up. It doesn't need "Specification" in the title. Many people will already add the word _specification_ as a descriptor when describing SMPS, so the "S" often becomes redundant.

**Desired State:**

- [x] Change this repository's name (drop "specification")
- [x] Remove the "S" in SMPS in the logo.
- [x] Update elsewhere....?

This PR fixes #80

**Notes for Reviewers**
I updated,
- README.md
- All images i.e. .png and .svg and dropped 'M' from SMPS
- All .yml and .html mentions of Service Mesh Performance Specification / SMPS to Service Mesh Performance / SMP
- Renamed files that had Service Mesh Performance Specification / SMPS in the name to Service Mesh Performance / SMP
- Refactored .html and .yml file to support the renamed/changed file 

**Needs Attention**
 - There are some more fixes that need to be done on this repo as they are not related to the Jekyll website. Someone should **refactor those codes to comply with project name** (Refactor Make, .go, and .proto).
-  [Layer5] (https://layer5.io/projects/service-mesh-performance-specification) will be fixed my a separate PR
 
**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   card: summary
   username: smp_spec
   url: "https://twitter.com/smp_spec"
-  image: "https://smp-spec.io/assets/img/logo/smps.png"
+  image: "https://smp-spec.io/assets/img/logo/smp.png"
 
 # logo:
-#   path: 
- # height: 80px #height in px, defaults to 52px
+#   path:
```
Comment (PluckyPrecious) [inline docs/_config.yml:17]: These could be modified too; lines 7, 12, 13, 14

**Comment by PluckyPrecious** [comment]
Welldone @JubayerJoy you did awesome with little time 💯

File: docs/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-description: "Service Mesh Performance Specification (SMPS): Measure your service mesh performance using SMPS. 
-SMPS provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments."
+description: "Service Mesh Performance (SMP): Measure your service mesh performance using SMP. 
+SMP provides a universal performance index to gauge your mesh’s efficiency against deployments in other organizations’ environments."
 author: Lee Calcote
 twitter: smp_spec
```
[the remark this answers is not in the document]
Comment (JubayerJoy) [reply docs/index.md]: Twitter handle changed. 👍

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 <p align="center"><i>If you’re using Meshery or if you like the project, please <a href="https://github.com/layer5io/meshery/stargazers">★</a> star this repository to show your support! 🤩</i></p>
```
[the remark this answers is not in the document]
Comment (JubayerJoy) [reply README.md]: Yeah! It should be SMP, I have made the change 😃

File: README.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ This specification is currently being advanced through the [Meshery](https://lay
 
 ## Example
 
-See the [sample results](docs/assets/spec/readme/example/service%20mesh%20performance%20specification%20result.yaml) file.
+See the [sample results](docs/assets/spec/readme/example/service%20mesh%20performance%20result.yaml) file.
```
[the remark this answers is not in the document]
Comment (JubayerJoy) [reply README.md]: I also think it should be changed, so made the change! 👍

File: docs/CNAME [hunk 5]
```diff
@@ -1 +0,0 @@
-smp-spec.io
```
[the remark this answers is not in the document]
Comment (JubayerJoy) [reply docs/CNAME:1]: @PluckyPrecious, Ops! excluded it for testing, fixed it. keeping it to "smp-spec.io" for now. 😃

**Comment by JubayerJoy** [comment]
Hi, @PluckyPrecious, there were some sign-off issues (Keeps happening 😭) in this PR, so made a new [PR](https://github.com/layer5io/service-mesh-performance-specification/pull/84) covering all the changes you have suggested. Please have a look 😃

File: README.md [hunk 6]
```diff
@@ -26,17 +26,17 @@ This specification is currently being advanced through the [Meshery](https://lay
 
 ## Example
 
-See the [sample results](docs/assets/spec/readme/example/service%20mesh%20performance%20result.yaml) file.
+See the [sample results](docs/assets/perf/readme/example/service%20mesh%20performance%20result.yaml) file.
 
-<a href="docs/assets/spec/readme/service mesh performance example.gif"><img src="docs/assets/spec/readme/service mesh performance example.gif" /></a>
+<a href="docs/assets/perf/readme/service mesh performance example.gif"><img src="docs/perf/spec/readme/service mesh performance example.gif" /></a>
```
Comment (PluckyPrecious) [inline README.md:31]: Hi @JubayerJoy you made a mistake here, it should be 

src="docs/assets/perf/readme/service mesh performance example.gif" /></a> 👍🏽

Comment (JubayerJoy) [reply README.md:31]: Wow! Your review skill is very impressive :clap:
I already have fixed in my latest [PR](https://github.com/layer5io/service-mesh-performance-specification/pull/84)

Thanks!

Comment (PluckyPrecious) [reply README.md:31]: Haha! 😆 Thanks a bunch. 🙌🏽

**Pull request closed without merging**